airflow/kubernetes/kube_client.py

```diff
@@ -125,5 +138,5 @@ def get_kube_client(
     if ssl_ca_cert:
         configuration.ssl_ca_cert = ssl_ca_cert
 
-    api_client = client.ApiClient(configuration=configuration)
+    api_client = client.ApiClient(configuration)
```

Comment:
Using keyword arguments is always recommended
```suggestion
    api_client = client.ApiClient(configuration=configuration)
```